lib/apiservers/portlayer/restapi/handlers/storage_handlers.go

```diff
@@ -273,37 +273,40 @@ func (handler *StorageHandlersImpl) RemoveVolume(storage.RemoveVolumeParams) mid
 	return storage.NewRemoveVolumeOK() //TODO: this is just a stub for now.
 }
 
-// convert an SPL Image to a swagger-defined Image
-func convertImage(image *spl.Image) *models.Image {
-	var parent, selfLink *string
+//VolumesList : Lists available volumes for use
+func (handler *StorageHandlersImpl) VolumesList(params storage.ListVolumesParams) middleware.Responder {
+	defer trace.End(trace.Begin("storage_handlers.VolumeList"))
```

Comment:
ah will do, that is in a lot of places... I will fix it here and file a GI about it a little later